go-types/shardrange/partition_range.go

```diff
@@ -25,6 +25,11 @@ func NewFullRange() Range {
 	}
 }
 
+// IsZeroed returns true if the begin and end are both 0
+func (r Range) IsZeroed() bool {
```

Comment:
nit: traditionally this is named `IsZero` ([example](https://golang.org/pkg/time/#Time.IsZero))